src/proxy_lib/CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ add_library(${PROJECT_NAME}::proxy ALIAS umf_proxy)
 
 target_link_directories(umf_proxy PRIVATE ${LIBHWLOC_LIBRARY_DIRS})
 
+target_compile_definitions(umf_proxy
+                           PRIVATE "-DUMF_SRC_VERSION=${UMF_SRC_VERSION}")
```

Comment:
why not ```PRIVATE UMF_SRC_VERSION=${UMF_SRC_VERSION}``` ?

Reply:
I think both should work

// we use this syntax currently: https://github.com/oneapi-src/unified-memory-framework/blob/main/src/CMakeLists.txt#L84

Reply:
Done